app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,19 @@
+!!!
+%html
+  %head
+    %title InstagramChallenge
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+
+  %body
+    - if user_signed_in?
+      =link_to 'Sign out', destroy_user_session_path, method: :delete
+
+    = alert
+    = notice
+
+    %div.container{:id => 'main-container'}
+      %div.row
+        %div.col-md-6.col-md-offset-3.text-center
```

Comment:
`%div.col-md-6.col-md-offset-3.text-center` can be written as `.col-md-6.col-md-offset-3.text-center` since `%div` is implicit